src/components/Navbar.js

```diff
@@ -1,27 +1,36 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Navbar, Nav, Button } from 'react-bootstrap'
+import React from 'react';
+import styled from 'styled-components';
+import { Navbar, Nav, Button } from 'react-bootstrap';
+import Logo from '../assets/VeriFactLogo.svg';
 
 export default (props) => {
-  return (
-    <CustomNavbar sticky='top' collapseOnSelect expand='md'>
-      <Navbar.Brand href='/' style={{ color: '#30323D', fontSize: '1.9rem' }}>SG VERIFACT</Navbar.Brand>
-      <Navbar.Toggle aria-controls='responsive-navbar-nav' />
-      <Navbar.Collapse id='responsive-navbar-nav' style={{ backgroundColor: 'white' }}>
-        <Nav className='mr-auto' />
-        <Nav style={{ alignItems: 'center' }}>
-          <Nav.Link style={{ color: '#30323D' }} href='login'>Log In</Nav.Link>
-          <Nav.Link style={{ color: '#30323D' }} href='signup'>
-            Sign Up
-          </Nav.Link>
-          <Nav.Link href='/askquestion'>
-            <CustomButton>Ask a Question</CustomButton>
-          </Nav.Link>
-        </Nav>
-      </Navbar.Collapse>
-    </CustomNavbar>
-  )
-}
+	return (
+		<CustomNavbar sticky="top" collapseOnSelect expand="md">
+			<Navbar.Brand href="/" style={{ color: '#30323D', fontSize: '1.9rem' }}>
+				SG VERIFACT
+				<img src={Logo} alt="VeriFact logo" height="25rem"s/>
```

Comment:
I think the logo mark should be on the left, which is customary for marks in order to visually anchor the words to the mark. Maybe with a very small padding between the mark and the words (5px?). We can run this by Yu Fei to see what he thinks if you disagree.

Reply:
I think it makes sense! Will make the changes